config/defaults.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
   taskclusterLogImage: 'taskcluster/livelog:v3',
   testdroidProxyImage: 'taskcluster/testdroid-proxy:0.0.6',
   balrogVPNProxyImage: 'taskclusterprivate/taskcluster-vpn-proxy:0.0.1',
+  dindImage: 'jonasfj/dind-service:v1.6',
```

Comment:
yeah, we should push it to `taskcluster/dind-service:v1.6` I didn't have access at the time.